doc/en/assert.rst

```diff
@@ -208,6 +208,7 @@ It is possible to add your own detailed explanations by implementing
 the ``pytest_assertrepr_compare`` hook.
 
 .. autofunction:: _pytest.hookspec.pytest_assertrepr_compare
+   :noindex:
```

Comment:
Yes, because it had the same directive elsewhere and Sphinx complained about which one to add to its main index.